.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
     - chmod 777 ~/onFailure.sh
     - chmod 777 ~/onSuccess.sh
 install:
-    - mvn dependency:resolve dependency:resolve-plugins -DskipTests=true -Dmaven.javadoc.skip=true -B -V
+    - mvn clean install dependency:resolve dependency:resolve-plugins -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
We can leave as it is right now, I just wonder if we would have any issue as the install phase will eventually puts the jars produced in the local maven repository and we got an issue during the execution of tests of the script phase (which could be a problem if Travis reuse that host to build for another lightblue's project). Do you also think that problem could happen too?